atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsDeleteBenchmarks.java

```diff
@@ -83,4 +90,11 @@ public Object allRangeDelete(ConsecutiveNarrowTable.RegeneratingCleanNarrowTable
         return doDeleteRange(table, 1);
     }
 
+    @Benchmark
+    @Threads(1)
+    @Warmup(time = 5, timeUnit = TimeUnit.SECONDS)
+    @Measurement(time = 30, timeUnit = TimeUnit.SECONDS)
```

Comment:
On current postgres I get something like

```
KvsDeleteBenchmarks.deleteAllTimestamps  com.palantir.atlasdb.performance.backend.PostgresKeyValueServiceInstrumentation@192.168.99.100:5432  sample    78  202748.219 ± 12694.104  us/op
```

The count is a bit low. I guess it will improve by a lot once we have #3379 though

Reply:
Yeah, I also expect it to run faster on a dedicated box with the postgres image